microprofile/messaging/core/src/main/java/io/helidon/microprofile/messaging/ProcessorMethod.java

```diff
@@ -45,7 +56,7 @@ class ProcessorMethod extends AbstractMessagingMethod {
     }
 
     @Override
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked", "checkstyle:methodlength"})
```

Comment:
do not suppress checkstyle warnings in code.
Use checkstyle-suppressions.xml instead if really needed

Reply:
supress removed